content/docs/0.8.x/integrations/migrate_from_070_to_080/index.md

```diff
@@ -0,0 +1,739 @@
+        }
+    return nil
+}
+
+func HandleGetSliTriggeredEvent(myKeptn *keptnv2.Keptn, incomingEvent cloudevents.Event, data *keptnv2.GetSLITriggeredEventData) error {
+	log.Printf("Handling get-sli.triggered Event: %s", incomingEvent.Context.GetID())
+	return nil
+}
+
+func HandleDeploymentTriggeredEvent(myKeptn *keptnv2.Keptn, incomingEvent cloudevents.Event, data *keptnv2.DeploymentTriggeredEventData) error {
+	log.Printf("Handling deployment.triggered Event: %s", incomingEvent.Context.GetID())
+	return nil
+}
+```
+
+*Note*: The full source code of those example is available in our [keptn-service-template-go](https://github.com/keptn-sandbox/keptn-service-template-go).
+
+### Sending CloudEvents
+
+While in keptn/go-utils 0.7.x you had to take care of sending CloudEvents in the correct format, we have tried to hide this complexity by introducing helper functions in keptn/go-utils 0.8.x.
+
+For instance, sending a deployment-finished event in 0.7.x looked like this:
+```golang
+
+	source, _ := url.Parse("helm-service")
+	contentType := "application/json"
+
+	var deploymentStrategyOldIdentifier string
+	if deploymentStrategy == keptnevents.Duplicate {
+		deploymentStrategyOldIdentifier = "blue_green_service"
+	} else {
+		deploymentStrategyOldIdentifier = "direct"
+	}
+
+	depFinishedEvent := keptnevents.DeploymentFinishedEventData{
+		Project:            keptnHandler.KeptnBase.Project,
+		Stage:              keptnHandler.KeptnBase.Stage,
+		Service:            keptnHandler.KeptnBase.Service,
+		TestStrategy:       testStrategy,
+		DeploymentStrategy: deploymentStrategyOldIdentifier,
+		Image:              image,
+		Tag:                tag,
+		Labels:             labels,
+		DeploymentURILocal: [],
+    	DeploymentURIPublic: []
+	}
+	
+
+	event := cloudevents.Event{
+		Context: cloudevents.EventContextV02{
+			ID:          uuid.New().String(),
+			Time:        &types.Timestamp{Time: time.Now()},
+			Type:        keptnevents.DeploymentFinishedEventType,
+			Source:      types.URLRef{URL: *source},
+			ContentType: &contentType,
+			Extensions:  map[string]interface{}{"shkeptncontext": keptnHandler.KeptnContext},
+		}.AsV02(),
+		Data: depFinishedEvent,
+	}
+
+	err := keptnHandler.SendCloudEvent(event)
+
+	if err != nil {
+        errMsg := fmt.Sprintf("Failed to send task finished CloudEvent (%s), aborting...", err.Error())
+        log.Println(errMsg)
+	}
+```
+
+With 0.8.x it looks like this:
+```golang
+    ServiceName := "helm-service"
+    
+	deploymentFinishedEventData := &keptnv2.DeploymentFinishedData{
+		DeploymentURIsLocal: ["http://a.b.c.d/foobar"],
+	}
+
+	_, err = myKeptn.SendTaskFinishedEvent(deploymentFinishedEventData, ServiceName)
+
+	if err != nil {
+		errMsg := fmt.Sprintf("Failed to send task finished CloudEvent (%s), aborting...", err.Error())
+		log.Println(errMsg)
+	}
+```
+
+### Processing Keptn CloudEvents requires sending a `.started` and `.finished` CloudEvent with `triggeredid` set to the original event id
```

Comment:
Please shorten it to: *Processing Keptn CloudEvents * 

and provide the long-form in a sentence below: `Processing Keptn CloudEvents requires sending a `.started` and `.finished` CloudEvent with `triggeredid` set to the original event id`